clients/admin-ui/src/features/system/ClassifySystemsTable.tsx

```diff
@@ -40,6 +59,7 @@ const ClassifySystemsTable = ({ systems }: { systems: System[] }) => {
                 _hover={{ bg: "gray.50" }}
                 cursor="pointer"
                 onClick={() => handleClick(system)}
+                hidden={hideEmpty && !classifyInstance?.has_labels}
```

Comment:
instead of using the `hidden` attribute (which I believe would still include the component in the DOM, just with a `hidden` attribute) it's more common to filter at the data level (to be data-driven!), then always have the DOM components render the same way (this way, they're not included in the DOM at all, since they weren't in the data). One way to do this would be something like, before the parent `return`, i.e. L36:

```typescript
  const filteredSystems = hideEmpty
    ? systems.filter(
        (system) => classifyInstanceMap.get(system.fides_key)?.has_labels
      )
    : systems;
```

and then you can keep the same `<Tr />` component that was already there, but replace the `systems` on L53 with `filteredSystems`

here's another example: https://github.com/ethyca/fides/blob/SteveDMurphy-400-toggle-system-classify/clients/admin-ui/src/features/system/SystemsManagement.tsx/#L19

Reply:
thank you @allisonking ! Going down that road now, I wasn't sure if there was a cost/benefit either way and happy to know that best path is what you are suggesting 🙌🏽